packages/contracts/contracts/universal/MultiSigWallet.sol

```diff
@@ -47,13 +52,13 @@ abstract contract MultiSigWallet is IMultiSigWallet, ReentrancyGuardUpgradeable
     uint256 public transactionCount;
 
     /**
-     * @notice Only allow this contract to call the functions.
-     *         This ensures that function is only executed through a multisig-based process.
+     * @notice Only allow specified governor contract to call the functions.
```

Comment:
AFAIK, the `GOVERNOR` in this contract should be the timelock contract of governor, right?
Since the timelock contract will call this contract.

Reply:
You're right, thanks for pointing that out. 
I think it's correct to name it governor, but the actual injection should be timelock proxy.

Reply:
I got your intention. Then, could you add this in the above comment (L20)?
```suggestion
     * @notice Only allow the specified governor contract to call the functions.
```